ensembl_rest.conf

```diff
@@ -0,0 +1,322 @@
+# This is the default configuration for Ensembl REST applications. Take your own copy
+# and make it available to Catalyst via the ENSEMBL_REST_CONFIG or CATALYST_CONFIG ENV vars.
+
+
+name = EnsEMBL::REST
+default_view = HTML
+
+site_name = Ensembl Rest API
+service_name = Ensembl REST API
+service_logo = /static/img/e-ensembl.png
+service_parent_url = http://www.ensembl.org
+wiki_url = http://github.com/Ensembl/ensembl-rest/wiki
+copyright_footer = <a href='http://www.sanger.ac.uk'>WTSI</a> &amp; <a href='http://www.ebi.ac.uk'>EBI</a>
+
+jsonp=1
+
+<Model::Registry>
+  ###### Database settings. Use if you want to connect to a single database instance. Common options are given below
+
+  host = ensembldb.ensembl.org
+  port = 3337
+  user = anonymous
```

Comment:
What's going on here? Are we still connecting to an external service in some cases?